File: Beecrowd/1144 Sequência Lógica/main.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
    int N, i;

    cin >> N;

    for (i = 1; i <= N; i++)
    {
        cout << i << " " << (i * i) << " " << (i * i * i) << endl;
        cout << i << " " << (i * i + 1) << " " << (i * i * i + 1) << endl;
    }
    return 0;
}

File: Beecrowd/1144 Sequência Lógica/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int n;

        n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            System.out.printf("%d %d %d%n", i, (int) Math.pow(i, 2), (int) Math.pow(i, 3));
            System.out.printf("%d %d %d%n", i, (int) (Math.pow(i, 2) + 1), (int) (Math.pow(i, 3) + 1));
        }

        sc.close();
    }
}